src/components/organisms/topbar/project/navigation.tsx

```diff
@@ -39,7 +40,9 @@ export const ProjectTopbarNavigation = () => {
 				const iconClassName = cn("group-hover:text-green-200  group-active:text-green-800", {
 					"text-green-200": isSelected,
 				});
-				const href = `/projects/${projectId}${item.path.replace("{deploymentId}", deploymentId || "")}`;
+				const computedPath = item.key === "assets" && storeActiveTab ? `/${storeActiveTab}` : item.path;
+
+				const href = `/projects/${projectId}${computedPath.replace("{deploymentId}", deploymentId || "")}`;
```

Comment:
Let's rewrite this not to use `string.replace` and will make it more elegantly in `src/constants/navigation.constnants.ts` also.
Maybe something like this:
```

				const getPath = () => {
					switch (item.key) {
						case "assets":
							return storeActiveTab ? `/${storeActiveTab}` : "/assets";
						case "sessions":
							return deploymentId ? `/deployments/${deploymentId}/sessions` : "";
						case "deployments":
							return "/deployments";
						default:
							return item.path;
					}
				};

				const href = `/projects/${projectId}${getPath()}`;
```

Reply:
changed